docs/docs/guides/js_apps/authwit.md

```diff
@@ -0,0 +1,117 @@
+---
+title: How to use authentication witnesses (authwit)
+---
+
+This page assumes you have authwit set up correctly in your contract. To learn how to do that, [go here](../smart_contracts/writing_contracts/authwit.md).
+
+For an introduction to authentication witnesses on Aztec, [read this explainer](../../aztec/concepts/accounts/authwit.md).
+
+## Import libraries
+
+These are all the libraries you might need for using authwits in Aztec.js:
+
+```typescript
+import {
+  computeAuthWitMessageHash,
+  computeInnerAuthWitHash,
+  computeOuterAuthWitHash,
+} from "@aztec/aztec.js";
+```
+
+You may not need all of these.
+
+## Publicly deploy accounts
+
+:::note
+This is only required if you are using authwits in public
+:::note
```

Comment:
```suggestion
:::
```

formatting is messed up 
![image](https://github.com/AztecProtocol/aztec-packages/assets/18372439/6205eb03-252a-4373-b07e-8d99d077e8ab)